mico-core/src/main/java/io/github/ust/mico/core/resource/ServiceInterfaceResource.java

```diff
@@ -91,35 +91,37 @@
     }
 
     @GetMapping(SERVICE_INTERFACE_PUBLIC_IP_PATH)
-    public ResponseEntity<List<String>> getInterfacePublicIpByName(@PathVariable(PATH_VARIABLE_SHORT_NAME) String shortName,
+    public ResponseEntity<MicoServiceInterfaceStatusResponseDTO> getInterfacePublicIpByName(@PathVariable(PATH_VARIABLE_SHORT_NAME) String shortName,
```

Comment:
This is a change of the API → Document it in the PR and inform the UI developers so they are able to add the necessary changes in the UI to this PR.
Also consider to inform the team via Slack after the merge into master that there was a change to the API.

Reply:
I agree, but these weren't my changes, this is the result of the merge with the master branch, so the changes should already been communicated. To be sure, I can do it as well.